.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - uses: ardalanamini/auto-changelog@v4
+      id: changelog
+      with:
+        commit-types: |
+          feat: New Features
+          fix: Bug Fixes
+          docs: Documentation
+          experimental: Experimental features
```

Comment:
Should be matched ?
https://github.com/webstudio-is/webstudio/blob/3fa4afa2691227a77ded5248a6b3cdfb05c18630/.github/workflows/lint-pull-request.yaml#L29-L41

Reply:
Most of these are not important to user and many we just don't use

Reply:
What it will do if chore etc will be in our PRs

Reply:
Put it into "Other changes" section